File: elias/Problema0004/problema0004.js
const prompt = require("prompt-sync")({ sigint: true });

var base = parseFloat(prompt("Digite o valor da base do Triângulo: "));
var altura = parseFloat(prompt("Digite o valor da altura do Triângulo: "));

var area = (base*altura)/2;

console.log("A área do Triângulo é: " + area);

File: elias/Problema0004/problema0004.java


import java.util.Scanner;

public class problema0004 {

    public static void main(String[] args) {

    Scanner input = new Scanner (System.in);

        float altura, base, area;
        System.out.println("Digite a altura do triangulo:");
        altura = Float.valueOf(input.nextLine());
        System.out.println("Digite a base do triangulo:");
        base = Float.valueOf(input.nextLine());
        area = (altura * base) /2;
        System.out.println("A area do triangulo é: "+area);
    }
}